CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+### [4.6.0] 2023-06-02
+
+## Changed
+
+- For a query, if there is missing params, we now either remove it, or raise an Exception in case `handle_errors` is True.
```

Comment:
This was already the case before... The Exception type has changed though

Reply:
ah yes... and i guess it's not necessary for the user to know that info right ?
https://github.com/ToucanToco/toucan-connectors/pull/1107/commits/8e5dc40

Reply:
It's necessary to know that the exception has changed, in case it was previously caught

Reply:
https://github.com/ToucanToco/toucan-connectors/pull/1107/commits/830675377d1b38bf0cc8ad1c8d6d901faa9f8239